inception/inception/README.md

```diff
@@ -0,0 +1,834 @@
+# Inception in TensorFlow
```

Comment:
Can you move this file so that it overwrites `inception/README.md`? (This creates a second copy of the readme.)